journal-of-clinical-neurophysiology.csl

```diff
@@ -6,21 +6,21 @@
     <id>http://www.zotero.org/styles/journal-of-clinical-neurophysiology</id>
     <link href="http://www.zotero.org/styles/journal-of-clinical-neurophysiology" rel="self"/>
     <link href="http://www.zotero.org/styles/journal-of-biological-regulators-and-homeostatic-agents" rel="template"/>
-    <link href="http://edmgr.ovid.com/jcnp/accounts/ifauth.htm" rel="documentation"/>
+    <link href="http://journals.lww.com/clinicalneurophys/_layouts/15/1033/oaks.journals/informationforauthors.aspx" rel="documentation"/>
     <author>
       <name>Patrick O'Brien</name>
       <email>obrienpat86@gmail.com</email>
     </author>
     <category citation-format="numeric"/>
     <category field="biology"/>
-    <eissn>1537-1603</eissn>
     <issn>0736-0258</issn>
+    <eissn>1537-1603</eissn>
     <updated>2017-06-25T12:04:18+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author">
     <names variable="author" suffix=". ">
-      <name et-al-min="19" et-al-use-first="19" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <name et-al-min="7" et-al-use-first="3" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```

Comment:
ah sorry,  you're right, I was wrong -- but the test checks for "greater or equal" when it should check for "greater", right?